libraries/Microsoft.Bot.Builder.AI.LUIS/V3/LuisRecognizer.cs

```diff
@@ -20,6 +20,7 @@ namespace Microsoft.Bot.Builder.AI.LuisV3
     /// <summary>
     /// A LUIS based implementation of <see cref="ITelemetryRecognizer"/> for the V3 endpoint.
     /// </summary>
+    [Obsolete]
     public class LuisRecognizer : LuisV2.ITelemetryRecognizer
```

Comment:
You can rip out all of the V3 stuff you want--it has not been published and was experimental.

Reply:
I think it was. It is on 4.6 branch thats way I kept existing classes

Reply:
Emilio could you double check this? If we can remove stuff, removing is nice

Reply:
@carlosscastro yes, it is in 4.6 release